client/src/Components/Mention.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+
+export default function Mention(props) {
+  return (
+    <Card style={{ height: "200px" }}>
```

Comment:
Small improvement: Notice that the card has a shadow by default, but the one on the mockup doesn't